documentation/modules/rn-2.1.adoc

```diff
@@ -0,0 +1,77 @@
+
+[discrete]
+=== Filtered must-gather data collection
+
+You can specify options for the `must-gather` tool that enable you to filter the data by namespace, migration plan, or VMs.
+
+[discrete]
+=== SR-IOV network support
+
+You can migrate VMs with a single root I/O virtualization (SR-IOV) network interface if the {virt} environment has an SR-IOV network.
+
+[id="known-issues-2.1_{context}"]
+== Known issues
+
+[discrete]
+=== Disk copy stage does not progress
+// to be fixed in 2.2
+The disk copy stage of a {rhv-short} VM does not progress and the {project-short} web console does not display an error message. link:https://bugzilla.redhat.com/show_bug.cgi?id=1990596[(*BZ#1990596*)]
+
+The cause of this problem might be one of the following conditions:
+
+* The storage class does not exist on the target cluster.
+* The VDDK image has not been added to the `HyperConverged` custom resource.
+* The VM does not have a disk.
+* The VM disk is locked.
+* The VM time zone is not set to UTC.
+* The VM is configured for a USB device.
++
+To disable USB devices, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#sect-Configuring_USB_Devices[Configuring USB Devices] in the Red Hat Virtualization documentation.
+
+To determine the cause:
+
+. Click *Workloads* -> *Virtualization* in the {ocp} web console.
+. Click the *Virtual Machines* tab.
+. Select a virtual machine to open the *Virtual Machine Overview* screen.
+. Click *Status* to view the status of the virtual machine.
+
+[discrete]
+=== VM time zone must be UTC with no offset
+
+The time zone of the source VMs must be UTC with no offset. You can set the time zone to `GMT Standard Time` after first assessing the potential impact on the workload. link:https://bugzilla.redhat.com/show_bug.cgi?id=1993259[(*BZ#1993259*)]
+
+[discrete]
+=== {rhv-short} resource UUID in custom resource causes a "Provider not found" error
+
+If a {rhv-short} resource is referenced by its UUID in a `Host`, `NetworkMap`, `StorageMap`, or `Plan` custom resource (CR), a "Provider not found" error is displayed. You must use the resource name. link:https://bugzilla.redhat.com/show_bug.cgi?id=1994037[(*BZ#1994037*)]
+
+[discrete]
+=== {rhv-short} resource name in custom resource might cause a critical "Ambiguous reference" condition
+
+If a {rhv-short} resource is referenced by name in a `NetworkMap`, `StorageMap`, or `Plan` CR and if a resource with the same name exists in another data center, the `Plan` CR displays a critical "Ambiguous reference" condition. You must rename the resource or use the resource UUID. link:https://bugzilla.redhat.com/show_bug.cgi?id=1993089[(*BZ#1993089*)]
+
+[id="technical-changes-2.1_{context}"]
+== Technical changes
+
+* The VMware Virtual Disk Development Kit (VDDK) SDK image is referenced in the `HyperConverged` custom resource. Before this update, it was referenced in the `v2v-vmware` config map.
```

Comment:
This is important. Can you please move it to the beginning of the RN?

Reply:
That would be an unusual structure. I'll try it and see how it looks.